openproblems/tasks/_batch_integration/batch_integration_graph/api.py

```diff
@@ -0,0 +1,34 @@
+from .datasets.immune import immune_batch
+
+
+def check_dataset(adata):
+    """Check that dataset output fits expected API."""
+
+    assert "X_uni" in adata.obsm
+    assert "batch" in adata.obs
+    assert "labels" in adata.obs
+    assert "uni_connectivities" in adata.obsp
+
+    return True
+
+
+def check_method(adata):
+    """Check that method output fits expected API."""
+    assert "connectivities" in adata.obsp
```

Comment:
Either check for "distances" as well, or remove this from the API description in the README.